pkg/kubelet/cm/container_manager_linux.go

```diff
@@ -328,8 +328,17 @@ func NewContainerManager(mountUtil mount.Interface, cadvisorInterface cadvisor.I
 
 	// Initialize CPU manager
 	if utilfeature.DefaultFeatureGate.Enabled(kubefeatures.CPUManager) {
+		var cpuManagerPolicyOptions map[string]string
+		if utilfeature.DefaultFeatureGate.Enabled(kubefeatures.CPUManagerPolicyOptions) {
+			cpuManagerPolicyOptions = nodeConfig.ExperimentalCPUManagerPolicyOptions
+		} else {
+			klog.InfoS("CPU Manager policy options feature gate not enabled, ignoring values", "options", nodeConfig.ExperimentalCPUManagerPolicyOptions)
+			cpuManagerPolicyOptions = nil
```

Comment:
Setting this to `nil` here is unnecessary. It's initial value in the declaration is nil.

Also -- I think we should only print this warning if `nodeConfig.ExperimentalCPUManagerPolicyOptions` has some values set in it. Otherwise, the warning has no real meaning.

Reply:
ack to both, thanks!